sdk/search/azure-search-documents/README.md

```diff
@@ -106,7 +106,7 @@ There are several types of operations that can be executed against the service:
 
 ### Create an index
 
-Create Index using `SearchIndexClient` create in above [section](Create-a-SearchIndexClient)
+Create Index using `${searchServiceClient}` create in above [section](#create-a-searchserviceclient)
```

Comment:
I don't understand what you are trying to express here.

Reply:
Oh. I understand.

```
Create an index using the `searchServiceClient` instantiated in "[Create a SearchServiceClient](#create-a-searchserviceclient)".
```

Reply:
Done.